azure_li_services/units/system_setup.py

```diff
@@ -228,6 +228,7 @@ def set_stonith_service(config):
     # target name. We also expect the discovery output to be in the
     # format: target_IP:port,target_portal_group_tag proper_target_name
     discovery_format = '.*:.*,.* .*'
+    MAX_TIMEOUT = '30'
```

Comment:
can we choose a name that add some scope, e.g ```udev_settle_timeout``` I also wondered why the capital letters ?

Reply:
I tried to give it the meaning of constant.
Fixed.